src/common.props

```diff
@@ -0,0 +1,20 @@
+<Project>
+  <PropertyGroup>
+    <TargetFrameworks>net451;netstandard1.6</TargetFrameworks>
+    <NetStandardImplicitPackageVersion>1.6.1-*</NetStandardImplicitPackageVersion>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+
+    <AssemblyOriginatorKeyFile>../../tools/Key.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
+
+    <RepositoryType>git</RepositoryType>
+    <RepositoryUrl>https://github.com/aspnet/javascriptservices</RepositoryUrl>
+
+    <GenerateNeutralResourcesLanguageAttribute>true</GenerateNeutralResourcesLanguageAttribute>
```

Comment:
These Generate*Attribute settings default to true so you can remove them, but you still need to set the values for the generate attributes by setting properties for `<Company>` and etc.

These are the default proeprties we set in all of our other aspnet projects.

```xml
    <Product>Microsoft ASP.NET Core</Product>
    <NeutralLanguage>en-US</NeutralLanguage>
    <Company>Microsoft Corporation.</Company>
    <Authors>Microsoft</Authors>
    <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
    <ProjectUrl>https://asp.net</ProjectUrl>
    <RequireLicenseAcceptance>true</RequireLicenseAcceptance>
    <Serviceable>true</Serviceable>
```

To clarify something...you don't see this in the build/common.props files in other github.com/aspnet projects, but instead you will see a PackageReference to "Internal.AspNetCore.Sdk". (see below).